### DISCUSSION TITLE
Welcome to ts-ucan Discussions!

### DISCUSSION THREAD

**Discussion by bgins** [ucan-wg/ts-ucan#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to hold long-form discussions with our community. We hope that you:
  * Ask questions about UCANs and this library
  * Share your ideas for this library 
  * Show and tell your apps that use UCANs
  * Welcome others and be open-minded. Remember that we build this community together 💪.

Comment below if you would like to introduce yourself or post to another category.

**Comment by matheus23** [comment]
Hi! :wave: 
I'm Philipp from Germany and part of the https://fission.codes team :)

Currently am working on getting better spec conformance into ts-ucan as well as a high-level API with some of the commonly needed things, e.g. a store where you can put all your UCANs, abstractions for verifying UCAN capabilities with customizable semantics and a builder API to make it easier to define your UCANs :)
